src/Action.hs

```diff
@@ -280,44 +280,39 @@ sendMailToRole role msg = do
     forM_ users $ \user ->
         sendMailToUser [] user msg
 
-phaseAction :: (ActionPersist m, ActionSendMail m) => Topic -> PhaseAction -> m ()
-phaseAction t = \case
-    JuryPhasePrincipalEmail ->
-        sendMailToRole Principal EmailMessage
-            { _msgSubject = "[Aula] Thema in der Prüfungsphase"
-            , _msgBody = ST.unlines $
-                  [ "Liebe Schulleitung,"
-                  , ""
-                  , "das Thema:"
-                  , ""
-                  , "    " <> topicTemplate
-                  , ""
-                  , "hat die Prüfungsphase erreicht und bedarf Ihrer Aufmerksamkeit."
-                  , ""
-                  , "hochachtungsvoll,"
-                  , "Ihr Aula-Benachrichtigungsdienst"
-                  ]
-            , _msgHtml = Nothing -- Not supported yet
-            }
-    ResultPhaseModeratorEmail ->
-        sendMailToRole Moderator EmailMessage
-            { _msgSubject = "[Aula] Thema in der Ergebnisphase"
-            , _msgBody = ST.unlines $
-                  [ "Liebe Moderatoren,"
-                  , ""
-                  , "das Thema:"
-                  , "    " <> topicTemplate
-                  , ""
-                  , "hat die Ergebnisphase erreicht und bedarf Ihrer Aufmerksamkeit."
-                  , ""
-                  , "hochachtungsvoll,"
-                  , "Ihr Aula-Benachrichtigungsdienst"
-                  ]
-            , _msgHtml = Nothing -- Not supported yet
-            }
-  where
-    topicTemplate = "the topic titled \"" <> t ^. topicTitle <> "\" at URL " <> absPath (listTopicIdeas t)
-    absPath p = "FIXME: http.../" <> absoluteUriPath (relPath p)  -- TODO!
+phaseAction :: (MonadReaderConfig r m, ActionPersist m, ActionSendMail m)
+            => Topic -> PhaseAction -> m ()
+phaseAction topic phaseAction = do
+    cfg <- asks (view getConfig)
```

Comment:
`cfg <- view getConfig` is enough since `view` already works with `MonadReader`. Also if I recall `MonadReaderConfig` exposes `viewConfig`, thus `cfg <- viewConfig`.